kickstart/kickstart.json

```diff
@@ -40,7 +41,7 @@
                 "name": "ExampleNodeApp",
                 "oauthConfiguration" : {
                     "authorizedRedirectURLs": ["http://localhost:3000/oauth-callback"],
-                    "logoutURL": "http://localhost:4200/",
+                    "logoutURL": "http://localhost:3000/",
```

Comment:
I like the quickstarts not being on port 3000 as that is where the astro site wants to run.  This could be moved to a variable and then replaced here and line 43 too.